src/spec/swagger-v11--submitted.json

```diff
@@ -2282,7 +2282,7 @@
         "nonce":{
           "type":"number",
```

Comment:
Shouldn't this be a string too?

Reply:
That one might be a `nonce` related to withdrawals themselves and maybe not related to auth

Reply:
I am not sure, this one is part of `withdrawalPayload` and I have seen another one in `starkOrder` payload, which is also number, but I only changed the ones that also have `signature` in body or query params, does that makes sense or that stark order nonce is also string?

Reply:
Honestly don't know, but if the task is auth only, then the PR is good enough for me :D